tests/water_test.py

```diff
@@ -0,0 +1,62 @@
+import jax_test3 as jt3
+import pytest as pt
+
+import water as w
+import numpy as np
+
+np.set_printoptions(linewidth=120)
+
+
+@pt.mark.parametrize(
+    "coordinates, charges, order",
+    [
+        (
+            np.array(
+                [
+                    [-0.10142170869456, -0.07509332372525, 0.0],
+                    [-0.05369405529541, 1.75540922326286, 0.0],
+                    [1.66333017194447, -0.56362368204899, 0.0],
+                ]
+            ),
+            np.array([8.0, 1.0, 1.0]),
+            1,
+        ),
+        (
+            np.array(
+                [
+                    [-0.10142170869456, -0.07509332372525, 0.0],
+                    [-0.05369405529541, 1.75540922326286, 0.0],
+                    [1.66333017194447, -0.56362368204899, 0.0],
+                ]
+            ),
+            np.array([8.0, 1.0, 1.0]),
+            2,
+        ),
+        (
+            np.array(
+                [
+                    [-0.10142170869456, -0.07509332372525, 0.0],
+                    [-0.05369405529541, 1.75540922326286, 0.0],
+                    [1.66333017194447, -0.56362368204899, 0.0],
+                ]
+            ),
+            np.array([8.0, 1.0, 1.0]),
+            3,
+        ),
+        (
+            np.array(
+                [
+                    [-0.10142170869456, -0.07509332372525, 0.0],
+                    [-0.05369405529541, 1.75540922326286, 0.0],
+                    [1.66333017194447, -0.56362368204899, 0.0],
+                ]
+            ),
+            np.array([8.0, 1.0, 1.0]),
+            4,
+        ),
+    ],
+)
+def test_dervs(coordinates, charges, order):
```

Comment:
You can simplify the parametrization of the test. The only parameter that can vary is the derivative order: the reference results are only available for the water molecule at one specific geometry.